sui_core/src/authority/authority_store.rs

```diff
@@ -834,10 +834,10 @@ impl<const ALL_OBJ_VER: bool, S: Eq + Serialize + for<'de> Deserialize<'de>>
         let (sequenced_to_write, schedule_to_write): (Vec<_>, Vec<_>) = ids
             .zip(versions.iter())
             .map(|(id, v)| {
-                let version = v.unwrap_or_else(SequenceNumber::new);
+                let version = v.unwrap_or_else(|| SequenceNumber::from(1));
```

Comment:
There is `OBJECT_START_VERSION` in object.rs, may be good to use that constant
Could you add comments here on why this must be 1?

Reply:
fixed